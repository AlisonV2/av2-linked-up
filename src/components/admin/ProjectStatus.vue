<template>
  <div class="row mb-4">
    <div class="col-12 status-description">
      <span class="project-bullet project-pending">&#9673;</span><span class="mx-2">Pending</span>
      <span class="project-bullet project-progress">&#9673;</span><span class="mx-2">In progress</span>
      <span class="project-bullet project-accepted">&#9673;</span><span class="mx-2">Accepted</span>
      <span class="project-bullet project-denied">&#9673;</span><span class="mx-2">Denied</span>
    </div>
  </div>
</template>

<script>
/**
 * @exports ProjectStatus
 * @type {Component}
 */
export default {
    name: 'ProjectStatus'
};
</script>

<style lang="scss">
.status-description {
  text-align: center;
}
</style>
