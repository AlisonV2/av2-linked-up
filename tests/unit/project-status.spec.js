import { shallowMount } from '@vue/test-utils';
import ProjectStatus from '@/components/admin/ProjectStatus';

/**
 * @module ProjectStatusTest
 */
describe('Categories.vue', () => {
  it('Categories Snapshot', () => {
    const component = shallowMount(ProjectStatus);
    expect(component.element).toMatchSnapshot();
  });

  it('Check if CategoriesGeoloc is rendered', () => {
    const component = shallowMount(ProjectStatus);
    const item = component.find('.project-bullet');
    expect(item.exists()).toBe(true);
  });
});
