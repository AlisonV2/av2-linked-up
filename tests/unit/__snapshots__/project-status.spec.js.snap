// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Categories.vue Categories Snapshot 1`] = `
<div
  class="row mb-4"
>
  <div
    class="col-12 status-description"
  >
    <span
      class="project-bullet project-pending"
    >
      ◉
    </span>
    <span
      class="mx-2"
    >
      Pending
    </span>
    <span
      class="project-bullet project-progress"
    >
      ◉
    </span>
    <span
      class="mx-2"
    >
      In progress
    </span>
    <span
      class="project-bullet project-accepted"
    >
      ◉
    </span>
    <span
      class="mx-2"
    >
      Accepted
    </span>
    <span
      class="project-bullet project-denied"
    >
      ◉
    </span>
    <span
      class="mx-2"
    >
      Denied
    </span>
  </div>
</div>
`;
